datadog-opentelemetry/src/core/configuration/configuration.rs

```diff
@@ -780,6 +780,12 @@ pub struct Config {
     /// A list of default tags to be added to every span
     /// If DD_ENV or DD_VERSION is used, it overrides any env or version tag defined in DD_TAGS
     global_tags: ConfigItem<Vec<(String, String)>>,
+    /// OTEL resource attributes parsed from OTEL_RESOURCE_ATTRIBUTES env var
+    otel_resource_attributes: ConfigItem<Vec<(String, String)>>,
+    /// OTEL metrics exporter type
+    otel_metrics_exporter: ConfigItem<Cow<'static, str>>,
+    /// OTEL metrics temporality preference
+    otel_metrics_temporality_preference: ConfigItem<Cow<'static, str>>,
```

Comment:
same remark about type conversion with the temporality preference, I think it's better to do validation and cast to an enum here.

The manual `set_otel_metrics_temporality_preference` method should take an enum type instead of a string for instance

Reply:
We were actually missing the `set_otel_metrics_temporality_preference` method. I added it to the programatic api for configurations.